core/os/os.h

```diff
@@ -282,8 +284,8 @@ class OS {
 	virtual bool set_environment(const String &p_var, const String &p_value) const = 0;
 
 	virtual String get_name() const = 0;
-	virtual List<String> get_cmdline_args() const { return _cmdline; }
-	virtual List<String> get_game_args() const { return _project_cmdline; }
+	virtual List<String> get_cmdline_args() const { return _command_parser->get_cmdline_args(); }
+	virtual List<String> get_game_args() const { return _command_parser->get_project_args(); }
```

Comment:
Could name it as `get_project_args` as well, see godotengine/godot-proposals#190. 🙂

Reply:
Done! thanks for the suggestion!